### DISCUSSION TITLE
Invalid TXT name for domain verification

### DISCUSSION THREAD

**Discussion by hayellis** [community/community#50094] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

I am unable to verify a Pages domain because an invalid TXT name.

My username is _ellis-_, meaning my verification sub-domain becomes '_github-pages-challenge-ellis-'.
The trailing hyphen makes it an invalid PQDN.

I'd rather not have to change my username 😢 

![image](https://user-images.githubusercontent.com/330758/225266684-0752c139-eaa9-4f29-a64e-8409ff8032e9.png)

**Comment by yoannchaudet** [comment]
@ellis- That's an unfortunate overlook and a bug. I am opening an issue so we can address that on our end. I don't have an ETA for that, but we will be looking at it as soon as we can.
